lib/modules/hosts/imgur.js

```diff
@@ -98,18 +93,18 @@ const imgur = {
 		let getExpandoInfo;
 
 		if (info.images && info.images.length) {
-			getExpandoInfo = this._handleAlbum(elem, info);
+			getExpandoInfo = this._handleAlbum(href, info);
```

Comment:
Might as well return these directly (no `await`) and get rid of `getExpandoInfo`